comps/cores/mega/gateway.py

```diff
@@ -4,24 +4,12 @@
 import base64
```

Comment:
shall you totally remove this file as comps shouldn't have gateway concept.

Reply:
shall we move `gateway.py`  to `service.py`, and rename the class `Gateway`  to ```Service` or `BaseService`?